satellite/satellitedb/database_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDatabase(t *testing.T) {
```

Comment:
Sharing some thoughts... please refer to https://github.com/storj/storj/blob/master/pkg/irreparabledb/irreparabledb_test.go. Liked the testDatabase() with all the tests totally agnostic to the kind of db used.